src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj

```diff
@@ -41,25 +41,16 @@
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
Here you do not need to remove any packages, as the test csproj does not depend on the common dependencies.  Sorry for the confusion.  The only changes in this file should be to the versions.